src/logEntryProcessing.test.ts

```diff
@@ -1,11 +1,150 @@
-import { isRequestLogEntry } from './logEntryProcessing';
+import {
+    fieldValue,
+    isRequestLogEntry,
+    lambdaRequestLogData,
+    parseLambdaLogLine,
+    parseMessageJson,
+    parseNodeLogFormat,
+    parseReportField
+} from './logEntryProcessing';
+import {log} from "util";
 
 test('AWS Lambda Function reports are identified as request log entries', () => {
-    const logLine = "REPORT RequestId: 6fcf3b62-bdcb-4fd1-a8ac-4388dfd4784c Duration: 803.21 ms Billed Duration: 804 ms Memory Size: 256 MB Max Memory Used: 219 MB"
+    const logLine = 'REPORT RequestId: b1f86b7f-67b8-45a8-926b-1699f0f7ccae\tDuration: 1028.81 ms\tBilled Duration: 1029 ms\tMemory Size: 1024 MB\tMax Memory Used: 220 MB\t';
```

Comment:
Would be good to add [eslint](https://github.com/guardian/configs/tree/main/packages/eslint-config-typescript) to the repo to prevent debate over single vs double quotes.

Reply:
Agreed, I will add this in a clean PR as I expect it's going to produce a big, messy diff!